src/model.js

```diff
@@ -0,0 +1,6 @@
+import falcor from '@graphistry/falcor';
```

Comment:
@trxcllnt I think I am going to merge this PR and start a new one w/ the `falcor-react-redux` integration as I like very much the idea of having an immutable async JSON store 😎